packages/flutter_tools/lib/runner.dart

```diff
@@ -82,17 +82,17 @@ Future<int> run(
       // to send the original error that triggered the crash report.
       firstError ??= error;
       firstStackTrace ??= stackTrace;
-      await _handleToolError(firstError, firstStackTrace, verbose, args, reportCrashes, getVersion);
+      await _handleToolError(firstError!, firstStackTrace, verbose, args, reportCrashes, getVersion);
     });
   }, overrides: overrides);
 }
 
 Future<int> _handleToolError(
-  dynamic error,
-  StackTrace stackTrace,
+  Object error,
+  StackTrace? stackTrace,
   bool verbose,
   List<String> args,
-  bool reportCrashes,
+  bool? reportCrashes,
```

Comment:
seems like this should be non-nullable (or optional with a default)? We just throw a bang on it later

Reply:
fixed